src/elf/sym.rs

```diff
@@ -94,9 +94,9 @@ pub fn st_visibility(other: u8) -> u8 {
 
 /// Is this information defining an import?
 #[inline]
-pub fn is_import(info: u8, value: u64) -> bool {
+pub fn is_import(info: u8, shndx: u64) -> bool {
```

Comment:
idly wondering if we should de-pub this along with the other functions; i wonder why they were made `pub`, could have been a historical accident or perhaps someone uses them for whatever reason; i don't see bingrep for example using it anywhere